tests/integration/test_interaction.py

```diff
@@ -110,56 +131,69 @@ def place_item(self, item_value, zone_id, action_key=None):
             self.move_item_to_zone(item_value, zone_id, action_key)
 
     def drag_item_to_zone(self, item_value, zone_id):
-        element = self._get_unplaced_item_by_value(item_value)
-        target = self._get_zone_by_id(zone_id)
+        element = self._get_item_by_value(item_value)
+        if zone_id is None:
+            target = self._get_item_bank()
+        else:
+            target = self._get_zone_by_id(zone_id)
         action_chains = ActionChains(self.browser)
         action_chains.drag_and_drop(element, target).perform()
 
     def move_item_to_zone(self, item_value, zone_id, action_key):
         # Get zone position
-        zone_position = self._get_zone_position(zone_id)
+        if zone_id is None:
+            zone_position = len(self.all_zones)
```

Comment:
@cahrens Yeah, this code is quite confusing.

In keyboard placement mode pressing the Tab key cycles focus between zones and the item bank, with the item bank being the last one to gain focus.

`zone_position` is not really the best variable name for what we are doing here, I will rename it to `tab_press_count` and add some comments to explain what the code is doing.